-> in: function_to_run() in test_simple_logger
  variable: color = red
  variable: number = one
<- out: function_to_run()
-> in: function_to_run() in test_simple_logger
  variable: color = red
  variable: number = two
<- out: function_to_run()
-> in: function_to_run() in test_simple_logger
  variable: color = red
  variable: number = brie
<- out: function_to_run()
*******************************************************************************************
Exception: AHHHHHH!
*******************************************************************************************
-> in: function_to_run() in test_simple_logger
  variable: color = blue
  variable: number = one
<- out: function_to_run()
-> in: function_to_run() in test_simple_logger
  variable: color = blue
  variable: number = two
<- out: function_to_run()
-> in: function_to_run() in test_simple_logger
  variable: color = blue
  variable: number = brie
<- out: function_to_run()
*******************************************************************************************
Exception: AHHHHHH!
*******************************************************************************************
